sig-scheduling/annual-report-2024.md

```diff
@@ -12,36 +12,39 @@
    - Governance and leadership changes
 -->
 
+- **The performance improvement**
+  - Have put a lot of effort into QueueingHint ([KEP-4247](https://github.com/kubernetes/enhancements/tree/master/keps/sig-scheduling/4247-queueinghint)), which enhances the scheduler's retrying efficiency.
+  - Introduced the asynchronous preemption ([KEP-4832](https://github.com/kubernetes/enhancements/tree/master/keps/sig-scheduling/4832-async-preemption)), which improves the scheduling throughput with the preemption.
+  - Improved [the internal scheduling performance test (scheduler-perf)](https://github.com/kubernetes/kubernetes/tree/master/test/integration/scheduler_perf) to cover more scenarios and alert us when the degradation is introduced.
+- **DRA**: Improved the DRA scheduling with the structured parameters ([KEP-4381](https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/4381-dra-structured-parameters#kube-scheduler)).
+
 2. Are there any areas and/or subprojects that your group needs help with (e.g. fewer than 2 active OWNERS)?
 
+None.
```

Comment:
This is WIP. Need to check with the owners at each project.

Here's the checklist, I checked the ones I own plus I know there are enough maintainers.

- [ ]    cluster-capacity
- [ ]    descheduler 
- [x]    kube-scheduler-simulator
- [x]    kube-scheduler-wasm-extension
- [x]    kueue
- [ ]    kwok
- [x]    scheduler
- [ ]    scheduler-plugins


@a7i @ingvagabund @wzshiming @Huang-Wei Hi, I just randomly picked up one person per project. Can you help me answer this question, do you have enough maintainers at your project?

Reply:
/hold

Until we get the answers from all.

Reply:
scheduler-plugins could really benefit from having another active maintainer.

Reply:
cluster-capacity has only a single maintainer. Though, all the changes are related to k8s rebases only.
@a7i descheduler has ~3 active maintainers? We get help from community members from time to time.

Reply:
KWOK doesn't have enough active maintainers, although not much has changed.

Reply:
Thanks all, added scheduler plugins and kwok here.

/unhold